posts/change-data-capture-cloudsql-postgres-with-apache-airflow-1.md

```diff
@@ -0,0 +1,1016 @@
+              catchup=False, # If True, it will start historically to start_date
+              template_searchpath=TEMPLATES_DIR, # Add this**
+              start_date=datetime(2020, 12, 29, 0, 0, 0),
+              default_args=default_args) as dag:
+    ```
+
+    With that, we can simply pass the `.sql` file path to our operators instead of the entire query.  
+
+3. Update the `get_schema` task to show the following:
+
+    ```python
+    gcs_schema_keypath = (
+        f"{gcs_root}/"
+        "schemas/"
+        "{{execution_date.year}}/"
+        "{{execution_date.month}}/"
+        "{{execution_date.day}}/"
+        f"{db_name}/"
+        f"{db_name}_schema_"
+        "{{ts_nodash}}.csv"
+    )
+
+    get_schema = CloudSqlCsvExportOperator(
+        task_id = 'get_schema',
+        gcp_conn_id = gcp_conn_id,
+        database_instance = db_instance_id,
+        project_id = gcp_project_id,
+        database_name = db_name,
+        gcs_bucket = gcs_bucket,
+        gcs_keypath = gcs_schema_keypath,
+        offload = offload_export,
+        params = {
+            "tables": tables
+        },
+        export_sql = 'cloud_sql_cdc/export_csv/get_schema.sql',
+        pool = 'cloudsql_operations',
+        trigger_rule = 'one_success'
+    )
+    ```
+
+    > *Notice here how we also used `execution_date` to construct the GCS keypath. You'll see another macro there as well `{{ts_nodash}}`, which is the equivalent to `execution_date.isoformat()` with special characters and spaces removed, useful for time-based object keys.*
+
+4. Change the  `export_table` tasks to show the following:
+
+    ```python
+    for table in tables:
+
+        gcs_raw_keypath = (
+            f"{gcs_root}/"
+            "raw/"
+            f"{table}/"
+            "{{execution_date.year}}/"
+            "{{execution_date.month}}/"
+            "{{execution_date.day}}/"
+            f"{table}_"
+            "{{ts_nodash}}.csv"
+        )
+
+        export_table = CloudSqlCsvExportOperator(
+            task_id = f'export_{table}',
+            gcp_conn_id = gcp_conn_id,
+            database_instance = db_instance_id,
+            project_id = gcp_project_id,
+            database_name = db_name,
+            gcs_bucket = gcs_bucket,
+            gcs_keypath = gcs_raw_keypath,
+            offload = offload_export,
+            export_sql = f'cloud_sql_cdc/export_csv/{table}.sql',
+            pool = 'cloudsql_operations',
+            trigger_rule = 'one_success'
+        )
+
+        get_schema >> export_table >> complete_export
+    ```
+
+    As mentioned above, all source code here is [available for download on github](https://github.com/astronomer/cdc-cloudsql-airflow-demo). Now it's time to check that everything is set up by using the render tool.
+
+## Step 11: Render Template Check
+
+In your terminal, you can use the Astro CLI to render all templating using the following format:
```

Comment:
```suggestion
In your terminal, you can use the Astro CLI to render all templating using the following command:
```

Reply:
@jwitz how about "templating using a command with the following format:"?
We don't want them to copy-paste that line.

Reply:
It's standard across our docs to use pointed brackets as placeholders in a command, I think that's self-explanatory here.